pkg/logs/input/container/scanner.go

```diff
@@ -47,28 +49,54 @@ func New(sources []*config.LogSource, pp pipeline.Provider, a *auditor.Auditor)
 		default:
 		}
 	}
-
 	return &Scanner{
 		pp:      pp,
 		sources: containerSources,
 		tailers: make(map[string]*DockerTailer),
 		auditor: a,
+		done:    make(chan struct{}),
```

Comment:
size 1?

Reply:
We need this call to be blocking to be sure that we are not scanning while stopping the tailers.